test/validate/service_territory_test.py

```diff
@@ -13,7 +13,7 @@
     "df_name,expected_rows",
     [
         ("summarized_demand_ferc714", 3_195),
```

Comment:
And we don't need to update the `summarized_demand_ferc714` because it's getting all the same respondent-years as before now, due to the left merge.  Right?

Reply:
exactly